redisgraph/client.py

```diff
@@ -159,3 +159,14 @@ def delete(self):
         Deletes graph.
         """
         return self.redis_con.execute_command("GRAPH.DELETE", self.name)
+    
+    def merge(self, pattern):
+        """
+        Merge a node or relation.
+        """
+
+        query = 'MERGE '
+        if object:
```

Comment:
`if object:` I'm not sure I fully understand this condition, can you please explain this line?